gst/tensor_filter/tensor_filter_tensorflow_lite.c

```diff
@@ -53,7 +69,13 @@ tflite_loadModelFile (const GstTensorFilter * filter, void **private_data)
   tflite_data *tf;
   if (filter->privateData != NULL) {
     /** @todo : Check the integrity of filter->data and filter->model_file, nnfw */
-    return 1;
+    tf = *private_data;
+    if (!strcmp (filter->prop.model_file,
```

Comment:
- Close if same, and
- Skip if they are different?

(```!strcmp```  --> ```strcmp``` ?? )

Reply:
oops... it's fixed :)